File: moyoutlet/OfferLightCollectionViewCell.m
//
//  OfferLightCollectionViewCell.m
//  moyOutlet
//
//  Created by Stepan Stepanov on 29.05.16.
//  Copyright © 2016 Stepan Stepanov. All rights reserved.
//

#import "OfferLightCollectionViewCell.h"

@implementation OfferLightCollectionViewCell


- (instancetype)initWithCoder:(NSCoder *)aDecoder
{
    self = [super initWithCoder:aDecoder];
    if (self)
    {
        // change to our custom selected background view

    }
    return self;
}

- (void)setBounds:(CGRect)bounds {
    [super setBounds:bounds];
    self.contentView.frame = bounds;
}

@end

File: moyoutlet/OfferLightCollectionViewCell.swift
//
//  OfferLightCollectionViewCell.swift
//  moyOutlet
//
//  Created by Stepan Stepanov on 17.06.17.
//  Copyright © 2017 Stepan Stepanov. All rights reserved.
//

import Foundation

class OfferLightCollectionViewCell: UICollectionViewCell {
    var  offerItem = OfferItem()
    
    @IBOutlet weak var cellView: UIView!
    @IBOutlet weak var priceView: UIView!
    @IBOutlet weak var priceLabel: UILabel!
    @IBOutlet weak var brandLabel: UILabel!
    @IBOutlet weak var offerImage: UIImageView!
    @IBOutlet weak var titleLabel: UILabel!
    
    @objc func configure() -> Void {
        let model = OfferItemViewConfig.init(model: self.offerItem)
        
        self.priceLabel.text = model.price
        self.brandLabel.text = model.brand
        self.titleLabel.text = model.name
        
    }
    
    override func awakeFromNib() {
        super.awakeFromNib()
        self.priceView.layer.cornerRadius = 5
        self.priceView.layer.masksToBounds = true
        self.priceView.clipsToBounds = true
        print("awakeFromNib")
    }
    
}
